webapp/sources/rudder/rudder-web/src/main/webapp/templates-hidden/components/ComponentNodeProperties.html

```diff
@@ -101,8 +101,13 @@
                      <button class="btn btn-xs btn-default btn-clipboard" data-clipboard-text="{{formatContent(property)}}" data-toggle='tooltip' data-placement='left' data-container="html" title="Copy to clipboard"><i class="ion ion-clipboard"></i></button>
                    </div>
                    <div ng-if="isEdited(property.name)">
-                     <textarea placeholder="Value" msd-elastic rows="1" class="form-control input-sm input-value" ng-model="editedProperties[property.name].new.value"></textarea>
-                     <small class="text-danger" ng-if="!editedProperties[property.name].new.isValid">JSON check is enabled, but the value format is invalid</small>
+                     <div class="col-lg-6">
```

Comment:
Why col-lg ? this mean that it wont apply on lower resolution ? in general we use col-sm-x instead

Reply:
if reducing the size of the window though it's ok

Reply:
no idea, I copy pasted from somewhere else in the page.

Reply:
(actually, there's a rationnal: in smaller screen, it may be more confortable to have one below the other, since JSON properties can become quite wide)

Reply:
even with `lg`, you get existing prop mixed before it goes one below the other: 
![image](https://user-images.githubusercontent.com/44649/83812875-901b6b80-a6bc-11ea-9035-7734292a4a5f.png)